omb_eregs/settings.py

```diff
@@ -107,6 +107,7 @@
 USING_SSL = env.get_credential('USING_SSL', 'TRUE').upper() == 'TRUE'
 SESSION_COOKIE_SECURE = USING_SSL
 CSRF_COOKIE_SECURE = USING_SSL
+CAS_RENEW = True
```

Comment:
Is this what makes MAX force users to log in again if they've logged out from the admin UI? If so, we might want a comment to that effect.